### PR TITLE
FJP starvation spec

### PR DESCRIPTION
**Pull request by johanandren**
Refs #26973

### CODE REVIEW

File: akka-actor-tests/src/test/scala/akka/dispatch/ForkJoinPoolStarvationSpec.scala [hunk 1]
```diff
@@ -10,8 +10,8 @@ import com.typesafe.config.ConfigFactory
 
 object ForkJoinPoolStarvationSpec {
   val config = ConfigFactory.parseString("""
+      |akka.actor.serialize-messages = off
```
Comment (johanandren) [inline akka-actor-tests/src/test/scala/akka/dispatch/ForkJoinPoolStarvationSpec.scala]: This took a lot of time. Doesn't explain why the test started failing when we switched FJP but should probably be done anyway.

Comment (helena) [reply akka-actor-tests/src/test/scala/akka/dispatch/ForkJoinPoolStarvationSpec.scala]: Do you know why it was there in the first place?

Comment (johanandren) [reply akka-actor-tests/src/test/scala/akka/dispatch/ForkJoinPoolStarvationSpec.scala]: We have it on in pretty much all test suites for some historical reason, to catch messages that aren't serializable I guess.

**Review by patriknw — APPROVED** [review]
LGTM, let's merge this first and follow up with other improvements separately

**Review by chbatey — APPROVED** [review]

**Pull request merged**